src/Draco.SourceGeneration/Templates/LspModel.sbncs

```diff
@@ -142,9 +144,8 @@ namespace Draco.Lsp.Model;
 {{end}}
 
 #nullable enable
-#pragma warning disable CS8618
+#pragma warning disable CS9042
 {{for $decl in Declarations}}
     {{declaration($decl)}}
 {{end}}
-#pragma warning restore CS8618
-#nullable restore
```

Comment:
Why do we never restore this?

Reply:
Well, this is at the end of the file, so `#nullable restore` doesn't do much